packages/atomic/src/components/facets/atomic-segmented-facet/atomic-segmented-facet.tsx

```diff
@@ -103,6 +103,7 @@ export class AtomicSegmentedFacet
       label: this.label,
       facetId: this.facetId!,
       element: this.host,
+      hasBreadcrumbs: false,
```

Comment:
I would still prefer not to register the segmented facets in the store. If we need to register them in the future, I would prefer to register them separately from normal facets, since part of the reason that normal facets are registered is so that they their element can be cloned inside the refine modal. I feel like the behaviour of facets vs segmented facets is too different to register them in the same place.

Reply:
So, do you think it would be better to implement a new buildSegmentedFacet function instead of using the buildFacet function that's currently available

Reply:
Okay! Let's just go ahead and filter out back in the breadbox & remove the new `hasBreadcrumbs` param, the rest should be good

```
({facetId}) => this.bindings.store.state.facets[facetId]
```